### PR TITLE
reviews

### PR DESCRIPTION
**Pull request by JakeColes04**

### CODE REVIEW

**Comment by Dy1anF** [comment]
review for Jake Coles 27961155

reviewed by Dylan Fellows 28031745

Your code runs without errors and is put together and structured well, the main feedback i have is as follows.

you have not followed the naming conventions well when instantiating objects in your code. Some examples of this are the private variable "die1" all the way to 3. this should be changed to fit into the global naming conventions. this is using camel case and putting an underscore before the variable name to signify it is private within the class, otherwise things will get confusing.  I also noticed that your dice cannot roll a 6, this logic error is in your die.cs when adding the parameters to the .Next(). (the second number is non inclusive)

The last thing id say is that you have a a lot of Console.Writlines in your game.cs. much like my own you could have used some formatting to keep them all in 1 Console.Writeline using $ and \n.

in conclusion the code runs very well without erorr and has a okay user-interface that informs the user how to exit the code.

**Comment by AshtonNich** [comment]
Jake Coles, COL27961155

Ashton Nicholls NIC27865888

Overall your code is very well written and i can see that you have had some issues with getting the dice to roll all of the available numbers to roll and with the testing class code.

Other than this your code does meet the requirements as it is very well encapsulated and also readable to the user, and as i was reading through it all.

for your die rolling you needed to have changed the random.Next(1,6) to (1,7) instead as it does not inlcude the last number.

although the testing class is not apparent to be implemented, your exception handling proves that there would be no errors no matter what the user inputted.

Im happy with the code with my main feedback being to change the roll.cs to roll all the numbers on the side of the dice.
